client/device/cloudfmc/fmcconfig/retry.go

```diff
@@ -49,6 +49,7 @@ func UntilCreateDeviceRecordSuccess(ctx context.Context, client http.Client, cre
 			ctx,
 			UntilTaskStatusSuccess(ctx, client, readInp),
 			retry.NewOptionsBuilder().
+				Message("waiting for create device record task to finish").
```

Comment:
Waiting for FMC device record to be created on CDO...

Reply:
done 👍